Add missing docs about sortable directive (#289)
neo4j/docs-graphql: modules/ROOT/pages/directives/schema-configuration/field-configuration.adoc

This means `actedIn` can be updated on an update, but it is no longer available on `create` operations.

== `@sortable`

This directive sets the availability of the input fields for sorting.

> It has one arguments:

Reviewer: ```suggestion
It has a single argument:
```

> * **byValue**: if disabled, this field is not available for sorting.

Reviewer: any default?

Author: yeah, defaults to `true`. I added this, but I've noticed we don't add the defaults in the other parameters of this page

=== Definition

[source, graphql, indent=0]
----
"""Instructs @neo4j/graphql to generate sorting inputs for this field."""
directive @sortable(byValue: Boolean! = true) on FIELD_DEFINITION
----

=== Usage

With this definition:

[source, graphql, indent=0]
----
type Movie @node {
    title: String!
    description: String! @sortable(byValue: false)
}
----

The following input type is generated for Movies:

[source, graphql, indent=0]
----
"""
Fields to sort Movies by. The order in which sorts are applied is not guaranteed when specifying many fields in one MovieSort object.
"""
input MovieSort {
  title: SortDirection
}
----

> This means that movies can be sorted by the title, but not by the description:

Reviewer: ```suggestion
Movies can be sorted by the title, but not by description:
```

This is valid:
[source, graphql, indent=0]
----
query {
  movies(sort: {title: DESC}) {
    title
  }
}
----

> This won't be valid:

Reviewer: ```suggestion
This is not valid:
```

[source, graphql, indent=0]
----
query {
  movies(sort: {description: DESC}) {
    title
  }
}
----

> If no fields are sortable, the type `MovieSort` won't be generated, and the `sort` input won't be avaiable.

Reviewer: ```suggestion
If no fields are sortable, the type `MovieSort` is not generated, and the `sort` input isn't available.
```
